packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.java

```diff
@@ -66,10 +65,13 @@ public ExpoImageView(ReactContext context, RequestManager requestManager, OkHttp
       mRequestManager
           .load(sourceToLoad)
           .apply(options)
-          .listener(eventsManager)
+          .addListener(eventsManager)
           .into(this);
       mRequestManager
           .as(BitmapFactory.Options.class)
+          // Remove the SVGSoftwareLayerSetter
+          // since we don't need it here
+          .listener(null)
```

Comment:
In fact it's `ExpoImageViewManager` which provides `ExpoImageView` with a preconfigured `RequestManager`. Changed the comment to not mention `SVGSoftwareLayerSetter` (since in fact we want to remove all the listeners that usually are related to image loading).